imi-frankfurt/dataelementhub.model
# #113 - Prevent multiple definitions of the same language when creatin…

**What's in the PR**
* When creating a namespace, check if only one definition per language is present. if this is not the case throw an exception.

**How to test manually**
* Check if creating/updating a namespace with two or more definitions of the same language throws an exception
* test with [rest](https://github.com/imi-frankfurt/dataelementhub.rest/pull/89)
* close #113
